pages/spotlights.js

```diff
@@ -0,0 +1,26 @@
+import Head from "next/head";
+import Link from "next/link";
+import MetaTags from "../components/Metatags.js";
+import HitLogo from "../components/HitLogo.js";
+import Spotlight from "../components/Spotlight/Spotlight.js"
+import React from "react"; 
+
+
+export default function Home() {
```

Comment:
Oh, I'm noticing we're doing this wrong on every other page too now but we may be better off naming this function name to be representative of the page. I.e. swap `Home()` with `Spotlights()`.

I don't know when we will need it... I read [the docs](https://nextjs.org/docs/basic-features/pages) and nothing really talks about that function's name... so, I suppose you could leave this and no harm would happen. 🤷‍♂️